### PR TITLE
DO NOT MERGE: [#128160407] Spike: password rotation pipeline

### PR DESCRIPTION
**Pull request by keymon**
## What

Create a pipeline that automates the process of rotating credentials.

## How to review

This a small implementation of the story [#128160407](https://www.pivotaltracker.com/n/projects/1275640/stories/128160407). Use this PR to comment on the code, but it is not intended to be merged.

Close the PR instead.

## Who can review

Anyone but  @keymon or @richardc

### CODE REVIEW

File: concourse/pipelines/create-bosh-cloudfoundry.yml [hunk 1]
```diff
@@ -2304,6 +2305,132 @@ jobs:
         params:
           release: pipeline-pool
 
+  - name: reset-cloudfoundry-passwords
```
Comment (HenryTK) [inline concourse/pipelines/create-bosh-cloudfoundry.yml]: If we run this job twice and forget the manual step of updating Grafana password in the UI, then we lose the old Grafana password and can't log in. I don't know what recovery of that password is possible, would have to look under the hood of Grafana.

**Comment by HenryTK** [comment]
Looks good to me. Despite the comment above I will probably close this pull request and leave it to implementors.

File: concourse/pipelines/create-bosh-cloudfoundry.yml (hunk 1 above)
Comment (keymon) [inline concourse/pipelines/create-bosh-cloudfoundry.yml]: True. We were aware of that, but we did not really solve it.

Comment (HenryTK) [inline concourse/pipelines/create-bosh-cloudfoundry.yml]: Agreed. I think because it is low risk and low severity we don't need to do anything about it.

**Pull request closed without merging**